lib/assets/javascripts/twine.js.coffee

```diff
@@ -315,7 +315,7 @@ setupAttributeBinding('innerHTML', 'unsafe-html')
 
 preventDefaultForEvent = (event) ->
   (event.type == 'submit' || event.currentTarget.nodeName.toLowerCase() == 'a') &&
-  Twine.getAttribute(event.currentTarget, 'allow-default') != '1'
+  Twine.getAttribute(event.currentTarget, 'allow-default') in ['false', false, 0, undefined, null]
```

Comment:
Not a big worry really, mostly yields a tiny perf benefit

I'm cool with this PR as written 👍